Description: serde -> non optional because the dep is ignored otherwise
Author: Ananthu C V <weepingclown@disroot.org>
Forwarded: not-needed
Last-Update: 2024-06-15
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -53,7 +53,6 @@
 [dependencies.serde]
 version = "1.0.171"
 features = ["derive"]
-optional = true
 default-features = false
 
 [features]
